src/main/java/appeng/worldgen/QuartzWorldGen.java

```diff
@@ -72,12 +72,12 @@ public void generate( final Random r, final int chunkX, final int chunkZ, final
 			return;
 		}
 
-		final double oreDepthMultiplier = AEConfig.instance.quartzOresClusterAmount * seaLevel / 64;
+		final double oreDepthMultiplier = AEConfig.instance().getQuartzOresClusterAmount() * seaLevel / 64;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Cast one of the operands of this division operation to a "double". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2184)